clients/src/main/java/org/apache/kafka/common/metrics/IntGaugeSuite.java

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Stolen from https://github.com/apache/kafka/pull/7588. I will remove it once the PR is merged.